ksp_plugin_adapter/planetarium_camera_adjuster.cs

```diff
@@ -21,7 +21,10 @@ public class PlanetariumCameraAdjuster : UnityEngine.MonoBehaviour {
     }
     var reference_rotation =
         (UnityEngine.QuaternionD)adapter.Plugin().CameraReferenceRotation();
+    double scale = adapter.Plugin().CameraScale();
     if (should_transfer_camera_coordinates) {
+      // Preserve the position when the plotting frame changes.
```

Comment:
s/position/scale/

Reply:
No, the position. We preserve the position by causing the scale correction to be 1.